pkg/platform/packet/template.go

```diff
@@ -228,6 +228,25 @@ provider "aws" {
   # "us-east-1": https://docs.aws.amazon.com/general/latest/gr/r53.html.
   region = "us-east-1"
 }
+
+{{- else if eq .Config.DNSProvider "cloudflare" }}
+module "dns" {
+  source = "../lokomotive-kubernetes/dns/cloudflare"
+
+  providers = {
+    cloudflare = cloudflare.default
+  }
+
+  cluster_name             = "{{ .Config.ClusterName }}"
+  controllers_public_ipv4  = module.packet-{{.Config.ClusterName}}.controllers_public_ipv4
+  controllers_private_ipv4 = module.packet-{{.Config.ClusterName}}.controllers_private_ipv4
+  dns_zone                 = "{{ .Config.DNSZone }}"
+}
+
+provider "cloudflare" {
+  version = "~> 2.0"
+  alias   = "default"
```

Comment:
I don't think we need this and `providers` block. We only have one instance of this provider, it will be automatically propagated to the module, as the module does not instantiate it either.

Reply:
It's gone now.